exercises/proverb/canonical-data.json

```diff
@@ -0,0 +1,94 @@
+{
+    "exercise": "proverb",
+    "version": "1.0.0",
+    "comments": [
+        "JSON doesn't allow for multi-line strings, so all expected outputs are presented ",
+        "here as arrays of strings. It's up to the test generator to join the ",
+        "lines together with line breaks. ",
+        "All cases expect the student to construct a proverb of the form ",
+        "'For want of an X, the Y was lost.', terminating with the stock phrase ",
+        "'And all for the want of a Z.'"
```

Comment:
Looks like `Z` is a special case for _one_ of the inputs - could be worth calling that out specifically in the description.md

Reply:
No? I don't understand your comment; all inputs of non-zero length terminate with that stock phrase.

Reply:
The description.md doesn't specifically draw any explicit connection between the last line and the first input. This especially is confusing if you are aware of the proverb IRL, where the inputs are always the same and the last line is always "And all for the want of a nail".

Going solely by the description, it seems like Z should always be "nail". Yes, the data implicitly makes this a requirement in a single case - but surely the data should be informed by the requirement (i.e. the description) rather than it introducing a requirement by itself?

What I mean to say is that by the description it is obvious that the inputs affect X and Y, but it is not obvious that they affect Z; Z is really X[0] so to speak.

I should have added this note against the description.md rather than this line I realise 😅

Reply:
I see now what you mean, though I'm not sure that I agree: the obvious terminating phrase, to me, would always be `f"And all for the want of a {input[0]}."`. Still, I don't mind adding a bit of text to the description to make that more clear.

Reply:
The `description.md` specifies the problem so I think you can just remove these 3 lines (8-10) from the `config.json`.